ironic/tests/unit/drivers/modules/drac/test_raid.py

```diff
@@ -948,6 +1152,13 @@ def test_create_configuration_with_predefined_number_of_phyisical_disks(
         physical_disks = self._generate_physical_disks()
         mock_list_physical_disks.return_value = physical_disks
         mock_commit_config.side_effect = ['42', '12']
+        mock_change_physical_disk_state.return_value = {
```

Comment:
Due to the changes recommend above, the above line should be changed to:
mock_client.change_physical_disk_state.return_value = {